README.md

```diff
@@ -76,6 +77,32 @@ providing an API to convert user-entered number and date strings - in their
 own format - into actual numbers and dates, and conversely, to format numbers
 and dates into that string format.
 
+Even if the application deals with Enligh loacle only, it may still likely need
```

Comment:
"deals only with the English locale, it may still need"

"may" and "likely" don't go well together, one is enough to indicate uncertainty.